config/config-api/src/main/java/com/thoughtworks/go/config/ExecTask.java

```diff
@@ -298,6 +298,18 @@ public String toString() {
                 '}';
     }
 
+    public Arguments getArguments() {
+       if (args.isEmpty()) {
+           return argList;
+       }
+       Arguments arguments = new Arguments();
+       for (String arg : args.split(" ")) {
```

Comment:
@ibnc [This is](https://github.com/gocd/gocd/blob/68f8bafb4fea5d132a9363c3fe39b6eea1628466/common/src/main/java/com/thoughtworks/go/domain/builder/CommandBuilder.java#L53) what GoCD does with arguments provided as "args".

Splitting by space will cause problems when using a task such as:

```
<exec command="ls" args="'file with space'" />
```